### DISCUSSION TITLE
Why fre:ac keeps storing every song in an individual folder?

### DISCUSSION THREAD

**Discussion by JIVS** [enzo1982/freac#298] [Help & Support]
Its incredibly annoying, how can I disable that?

Sorry if its a dumb question but I can't find the option for that anywhere

**Comment by enzo1982** [comment]
You can set the pattern for naming output files in the config dialog (at the bottom of the initial config dialog page).

The default naming pattern is `<albumartist> - <album>/<track> - <artist> - <title>`, which will put output files in a folder named after the artist and album they belong to. This is great when you are converting whole albums, but when you have a bunch of files all belonging to different albums, most of the output files will end up in separate folders.

In this situation, you can set the output filename pattern to e.g. `<artist> - <album> - <track> - <title>`, simply `<artist> - <title>` or even `<filename>`. The last one will keep the filenames of the original files.

For more information on output filename patterns and the available placeholders, see [here](https://freac.org/manual/en/howto.html#anchor_pat).

**Reply by JIVS** [reply]
Just to clarify you mean I have to put the same album (eg: "music") on every file? manually?

I'm using the GUI version BTW.

**Reply by enzo1982** [reply]
No, you should change the _Filename pattern_ setting to not create a subfolder for each album. See the screenshot for where to find the setting. Then set this to one of those I suggested in my initial answer.

![Image2](https://user-images.githubusercontent.com/7796713/163849749-a95df610-9511-4aac-ba92-37ed6708f99c.png)
